src/scenes/AcceleratorRouter/Applications/ApplicationsList.tsx

```diff
@@ -60,10 +125,15 @@ const ApplicationList = () => {
         result.data
           .filter((application) => application.status !== 'Not Submitted')
           .map((application) => ({
+            // TODO: get country code from API search results
+            countryCode: '',
```

Comment:
@nickgraz and/or @sgrimm, we need to display a country value in the table of applications on the console side, but currently there's no data for country being returned in `ApplicationPayload`. Can we add/include either the country code or name to the payload? If not, what other data can be used to determine country?

Reply:
It looks like the "project accelerator details" are [filled in](https://github.com/terraware/terraware-server/blob/main/src/main/kotlin/com/terraformation/backend/accelerator/ApplicationService.kt#L54) when an application passes the pre-screen, which includes the country code, determined by the project boundary.

Reply:
@sgrimm do you know if the country is collected during the application process? Or is it only available after the pre-screen is passed?

Reply:
It looks like [this](https://github.com/terraware/terraware-web/blob/8d690c8601acb67644193b30be55540b61c0c184/src/services/ApplicationService.ts#L44) is the underlying API call that the view is making to populate the table. It is a custom endpoint (as opposed to the Search API) so we will need to add the data in there on the backend [here](https://github.com/terraware/terraware-server/blob/main/src/main/kotlin/com/terraformation/backend/accelerator/api/ApplicationsController.kt#L145-L157)

Reply:
Scratch that, the country code is stored directly on the `projects` row. So I think you can add `APPLICATIONS.projects.COUNTRY_CODE` to [this select](https://github.com/terraware/terraware-server/blob/main/src/main/kotlin/com/terraformation/backend/accelerator/db/ApplicationStore.kt#L603-L609) to get what you're looking for, since that fetch is what runs for the "list applications" API. It isn't guaranteed to have a value though